markdown/org/docs/sewing/knit-binding/en.md

```diff
@@ -49,11 +50,17 @@ A knit binding is less common, but often used on undershirts (as in, underwear).
 
 Whether you're using a knit band or a knit binding finish, you are going to have to stretch your binding fabric for good results. That is, assuming you're not sewing a straight seam. Here's why:
 
+<Note>
+
+The remainder of this documentation uses the term _binding_ to refer to both knit bands and knit binding.
```

Comment:
I think it's fine to use only one term, since it makes the document easier to read, but 'binding' and 'band' are not the same thing. I suggest rephrasing to "The remainder of this documentation refers to _binding_, but applies equally to knit bands."

Reply:
I changed the wording as suggested.